src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveClimateControlScheduleCommandClass.java

```diff
@@ -61,10 +72,80 @@ public void handleApplicationCommandRequest(SerialMessage serialMessage, int off
                 this.getVersion());
         int command = serialMessage.getMessagePayloadByte(offset);
         switch (command) {
+            case SCHEDULE_CHANGED_GET:
+                logger.debug("Answering with noop SCHEDULE_CHANGED_REPORT");
```

Comment:
Please update the logging to use the standard format with the NODE {} on the start